frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -140,6 +142,15 @@ export const getOCSRequestData = (
     },
   };
 
+  if (kms) {
+    requestData.spec.security = {
+      kms: {
+        connectionDetailsConfigMap: 'ocs-vault-connection-details',
+        tokenSecretName: 'ocs-kms-vault-token',
```

Comment:
I would prefer to make `ocs-kms-vault-token` a constant, since used at multiple places.